runtime/src/main/scala/com/soundcloud/twinagle/ServerBuilder.scala

```diff
@@ -5,16 +5,23 @@ import com.twitter.finagle.{Filter, Service}
 
 case class ServerBuilder(
     extension: EndpointMetadata => Filter.TypeAgnostic = _ => Filter.TypeAgnostic.Identity,
-    endpoints: Seq[ProtoRpc] = Seq.empty
+    endpoints: Seq[ProtoRpc] = Seq.empty,
+    prefix: String = "/twirp"
 ) {
 
   def register[T: AsProtoService](svc: T): ServerBuilder = {
     val protoService = implicitly[AsProtoService[T]].asProtoService(svc)
     this.copy(endpoints = endpoints ++ protoService.rpcs)
   }
 
+  def withPrefix(prefix: String): ServerBuilder = {
+    require(prefix.startsWith("/"), "prefix must start with slash")
+    require(!prefix.endsWith("/"), "prefix must not end with slash")
```

Comment:
Should we move these checks out of this method? Otherwise I can just do `ServerBuilder(prefix = "badpath/")` or similar and the validation won't catch it.